### ISSUE TITLE
accessibilitysupport_form button should be labeled with "send"

### ISSUE THREAD

**Issue by whuml** [moodle-an-hochschulen/moodle-theme_boost_union#830]
**Improvement of accessibility support form**
Some Boost Union Users noticed wrong meaning of button label "submit":

accessibilitysupport_form:
$this->add_action_buttons(true, get_string('submit'));
german translation: "submit" usually is translated as "speichern"

But should be translated here as: "senden"

**Describe alternatives you've considered**
As translation should be adjusted only for accessibilitysupport_form it might be necessary to implement ?

$this->add_action_buttons(true, get_string('send', 'theme_boost_union'));
"send" translated as "senden"
